[2022-01-05 12:23:32,876] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: generate_data.generate_data 2022-01-05T12:23:12.727896+00:00 [queued]>
[2022-01-05 12:23:32,923] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: generate_data.generate_data 2022-01-05T12:23:12.727896+00:00 [queued]>
[2022-01-05 12:23:32,924] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-05 12:23:32,924] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-01-05 12:23:32,925] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-05 12:23:32,963] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): generate_data> on 2022-01-05T12:23:12.727896+00:00
[2022-01-05 12:23:32,967] {standard_task_runner.py:52} INFO - Started process 46 to run task
[2022-01-05 12:23:32,972] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'generate_data', 'generate_data', '2022-01-05T12:23:12.727896+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/geanrate_data.py', '--cfg-path', '/tmp/tmpwznkmn6q', '--error-file', '/tmp/tmphh6absz0']
[2022-01-05 12:23:32,973] {standard_task_runner.py:77} INFO - Job 7: Subtask generate_data
[2022-01-05 12:23:33,097] {logging_mixin.py:104} INFO - Running <TaskInstance: generate_data.generate_data 2022-01-05T12:23:12.727896+00:00 [running]> on host c2aa36a26221
[2022-01-05 12:23:33,348] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=generate_data
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-05T12:23:12.727896+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-05T12:23:12.727896+00:00
[2022-01-05 12:24:08,467] {local_task_job.py:146} INFO - Task exited with return code Negsignal.SIGKILL
